packages/react-microlink/src/components/Card/CardMedia/Video/index.js

```diff
@@ -1,10 +1,16 @@
+import {Motion, spring} from 'react-motion'
 import React, {Component} from 'react'
 import styled from 'styled-components'
 
-import {getUrlPath} from '../../../../utils'
-
 import {PlayButton, ProgressBar} from './controls'
 import MediaWrap from '../wrap'
+import {getUrlPath} from '../../../../utils'
+
+const CustomMotion = styled(Motion)`
+div.microlink_card:not(:hover) & {
```

Comment:
My previous comment about not using `div` would apply to this, @Kikobeats

Reply:
actually this code is not necessary! removed